### DISCUSSION TITLE
How to use the second video card for training?

### DISCUSSION THREAD

**Discussion by NaughtDZ** [d8ahazard/sd_dreambooth_extension#315] [Q&A] [answered]
My GPU0 is rtx3070 8G, and my GPU1 is Tesla M40 24G.I don't want two graphics cards to work in parallel., I just want to only use the second video card, the Tesla M40, for training. I have added -- device id 1 to the startup item of WebUI, but when this plug-in runs, it still uses GPU0, RTX3070.

**Comment by NaughtDZ** [comment]
Can you add a listbox to chose which device to use ?
Dreambooth support "--gpus 0" or "--gpus "1" commend

**Comment by Lukium** [comment, marked as answer]
edit the webui-user.bat as follows:

after `@echo off` add:
```
set CUDA_DEVICE_ORDER=PCI_BUS_ID
set CUDA_VISIBLE_DEVICES=1
```
in `COMMAND_LINE_ARGS=` use `--device-id 0`

This will result in GPU 1 (your second GPU) being the only GPU seen by the UI, which means that it becomes --device-id 0 as far as it's concerned.

If you're on Linux, basically same thing just slightly different language (webui-user.sh):

```
export CUDA_DEVICE_ORDER=PCI_BUS_ID
export CUDA_VISIBLE_DEVICES=1
```
somewhere in the beginning I usually put it near the venv section

**Reply by NaughtDZ** [reply]
thx
